dashboard/app/views/reports/search_for_teachers.html.haml

```diff
@@ -0,0 +1,40 @@
+%center
+  %h1 Search for Teachers
+
+%br/
+
+%form{:method => "get"}
+  %table
+    %tr
+      %td
+        Email Filter
+      %td
+        %input{:name => "emailFilter", :id => "emailFilter"}
+    %tr
+      %td
+        Address Filter
+      %td
+        %input{:name => "addressFilter", :id => "addressFilter"}
+
+  %br/
+
+  %center
+    %button
+      = link_to "Search", :search_for_teachers
+
+%br/
+
+%h2 Teachers
+%table.teachers
+  - @teachers.each do |teacher|
+    %tr
+      %td
```

Comment:
just fyi, I don't think we prefer one way or another, but if a haml tag has just one thing in it you can one-line it:
````
%td= teacher.id
%td= teacher.name
````